lib/task-data/tasks/bootstrap-ubuntu.js

```diff
@@ -7,12 +7,12 @@ module.exports = {
     injectableName: 'Task.Linux.Bootstrap.Ubuntu',
     implementsTask: 'Task.Base.Linux.Bootstrap',
     options: {
-        kernelFile: 'vmlinuz-3.16.0-25-generic',
-        initrdFile: 'initrd.img-3.16.0-25-generic',
+        kernelFile: 'vmlinuz-3.19.0-56-generic',
+        initrdFile: 'initrd.img-3.19.0-56-generic',
         kernelUri: '{{ api.server }}/common/{{ options.kernelFile }}',
         initrdUri: '{{ api.server }}/common/{{ options.initrdFile }}',
-        basefs: 'common/base.trusty.3.16.0-25-generic.squashfs.img',
-        overlayfs: 'common/discovery.overlay.cpio.gz',
+        basefs: 'common/base.trusty.3.19.0-56-generic.squashfs.img',
+        overlayfs: 'common/discovery.3.19.0-56-generic.overlay.cpio.gz',
```

Comment:
Talking about refactoring, I suggest more. the URI/URL could be independent with kernel/initrd/basefs/overlayfs so that the static file server could be independent, it's very useful when a separate service for static file is used(also may benefit to RackHD in switch),  so just like this.
{
   ***
   baseUri: {{api.server}/common
   kernelFile: ***
   initrdFil: ***
   basefs: ***
   overlayfs: ***
   ***
}
and for ipxe profile https://github.com/RackHD/on-http/blob/master/data/profiles/linux.ipxe also need to be refactored, 
kernel <%=baseUri%>/<%=kernelFile%>
initrd <%=baseUri%>/<%=initrdFile%>
imgargs imgargs <%=kernelFile%> initrd=<%=initrdFile%> ****** BASEFS=<%=baseUri%>/<%=basefs%> OVERLAYFS=<%=baseUri%>/<%=overlayfs%> ***

so the hardcoded 'http' could be removed from 
https://github.com/RackHD/on-imagebuilder/blob/master/roles/initrd/provision_initrd/files/local#L109-L121

I know this is out of the original purpose of this PR, so it could be refactored in another PR,  so still :+1:  from me for this PR with another PR refactoring